assets/lib/locales/locales.php

```diff
@@ -112,7 +112,7 @@ public function numbers_for_index( $index, $how_many = 3, $test_up_to = 1000 ) {
 	public function index_for_number( $number ) {
 		if ( ! isset( $this->_index_for_number ) ) {
 			$gettext = new Gettext_Translations;
-			$expression = $gettext->parenthesize_plural_exression( $this->plural_expression );
+			$expression = $gettext->parenthesize_plural_expression( $this->plural_expression );
```

Comment:
@pedro-mendonca How about this one?

Reply:
> This method has been deprecated. Use [Plural_Forms()](https://developer.wordpress.org/reference/classes/plural_forms) instead.

Reply:
But only for v6.5

Reply:
The `locales.php` file is a clone from GlotPress. This should be reported upstream.